adoc/code/graph.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2025 The Khronos Group, Inc.
+// SPDX-License-Identifier: MIT
+
+#include <iostream>
+#include <sycl/sycl.hpp>
+#include <utility>
+using namespace sycl;  // (optional) avoids need for "sycl::" before SYCL names
+
+constexpr size_t N = 100;
+constexpr size_t Iter = 10;
+
```

Comment:
```suggestion

// This example illustrates a simplified iterative 1D solver
// which is recorded into a graph and replayed.
// Because input and output arrays are swapped after each iteration,
// we record two iterations into an immutable graph that
// can be executed consecutively. 
```